packages/generator/templates/app/blitz.config.ts

```diff
@@ -7,6 +7,15 @@ const config: BlitzConfig = {
       isAuthorized: simpleRolesIsAuthorized,
     }),
   ],
+  template:{
+    typeToComponentMap: {
```

Comment:
I found this comment from the previous PR: https://github.com/blitz-js/blitz/pull/2134#issuecomment-808513567

Was it discussed for this PR as well (I might have missed it)? If not, that seems like something we can create an issue for afterwards.